framework/web/Request.php

```diff
@@ -1753,6 +1871,17 @@ public function setUploadedFiles($uploadedFiles)
         $this->_uploadedFiles = $uploadedFiles;
```

Comment:
Do we need it public?

Reply:
See https://github.com/yiisoft/yii2/pull/15416#discussion_r158818756